modules/loaders/flake.nix

```diff
@@ -14,6 +14,12 @@ in
               type = lib.types.string;
               default.value = "flake.nix";
             };
+
+            inputs = lib.options.create {
+              description = "Inputs to replace in the loaded flake.";
+              type = lib.types.attrs.of lib.types.raw;
```

Comment:
The type of each input needs to be raw so we don't traverse into it looking for module system helpers.